xbmc/cores/AudioEngine/Encoders/overrides/android/AEEncoderFFmpegAndroid.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+/*
+ *      Copyright (C) 2010-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#if defined(TARGET_ANDROID)
+#include "../../AEEncoderFFmpeg.h"
+
+class CAEEncoderFFmpegAndroid: public CAEEncoderFFmpeg
+{
+public:
+  CAEEncoderFFmpeg* CreateInstance()
+  {
+    return new CAEEncoderFFmpegAndroid();
+  }
+  virtual CAEEncoderFFmpegAndroid() {};
+  virtual ~CAEEncoderFFmpegAndroid() {};
```

Comment:
should be possible to omit both ctor/dtor. virtual destructor is inherited from the base. if you want to keep the dtor, i'd mark it override.